pkg/parse/parse.go

```diff
@@ -959,7 +959,7 @@ func defaultIsolation() (define.Isolation, error) {
 func IsolationOption(isolation string) (define.Isolation, error) {
 	if isolation != "" {
 		switch strings.ToLower(isolation) {
-		case "oci":
+		case "oci", "default":
```

Comment:
based on your changes above, is "default" possible?

Reply:
I wonder the same, but otherwise LGTM.